packages/perseus/src/util/parse-perseus-json/regression-tests/data/radio-missing-noneOfTheAbove.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Thinking out loud... I wonder if we want to attribute the source of this item? Perhaps not. We can always run a full regression pass to see if it's safe to remove some (old) parsing logic. 🤔

Reply:
Doh, I see you and Mark basically discussed this above, I missed that. 

One option could be to use a JSONC parser (https://www.npmjs.com/package/jsonc-parser) which is built by Microsoft. That does start to diverge from how our production code does things though. 

I think the file naming convention is likely to serve us just fine for now.